packages/@aws-cdk/aws-sns/lib/topic-base.ts

```diff
@@ -80,8 +80,8 @@ export abstract class TopicBase extends Resource implements ITopic {
   /**
    * Subscribe some endpoint to this topic
    */
-  public addSubscription(subscription: ITopicSubscription): Subscription {
```

Comment:
making a change from `subscription` to `topicSubscription` actually can cause impact this actually breaks in python if you are using named arguments because this changes the argument from `subscription` to `topic_subscription`